components/registry-facade/pkg/registry/manifest.go

```diff
@@ -362,6 +371,13 @@ func DownloadManifest(ctx context.Context, fetcher remotes.Fetcher, desc ociv1.D
 	}
 
 	cfg = &res
+
```

Comment:
looks like this block isn't necessary, though haven't tested without the changes. (log output is a bit messy as i assumed support for `%s`):
```
{"level":"debug","message":"Before returning downloaded manifest, updated MediaType from '%s' to '%s'application/vnd.docker.distribution.manifest.v2+jsonapplication/vnd.docker.distribution.manifest.v2+json","serviceContext":{"service":"registry-facade","version":""},"severity":"DEBUG","time":"2021-03-09T17:19:37Z"}
```